docs/components/Popover/Popover.stories.mdx

```diff
@@ -0,0 +1,59 @@
+import { Meta } from "@storybook/addon-docs";
+
+<Meta title="Components/Overlays/Popover/Docs" />
+
+# Popover
+
+A Popover displays floating informative and actionable content positioned in
+relation to a target. The Popover can contain content, media, or other
+components.
+
+## Usage guidelines
+
+Some scenarios for Popover include the following:
```

Comment:
In the old version of the doc, this sentence was followed by the examples for "Informational" and "Functional" -- it seems weird to have this sentence and then nothing before the next Related Components section. But, I see that the old "Functional" section didn't actually have an example... 
So, maybe it is still worth including the Informational heading and text explanation here, and linking to the Informational Example?

Reply:
whoops, I goofed here. I like this idea, I'll add the explanation linking to the example 👍